pkg/controller/certificate-shim/sync_test.go

```diff
@@ -1128,10 +1128,12 @@ func TestSync(t *testing.T) {
 							Protocol: "HTTPS",
 							TLS: &gwapi.GatewayTLSConfig{
 								Mode: ptrMode(gwapi.TLSModeTerminate),
-								CertificateRef: &gwapi.LocalObjectReference{
-									Group: "core",
-									Kind:  "Secret",
-									Name:  "example-com-tls",
+								CertificateRefs: []*gwapi.SecretObjectReference{
+									{
+										Group: func() *gwapi.Group { g := gwapi.Group("core"); return &g }(),
+										Kind:  func() *gwapi.Kind { k := gwapi.Kind("Secret"); return &k }(),
+										Name:  "example-com-tls",
+									},
```

Comment:
suggestion: add a test case where `group` is left empty

Reply:
👍 Good idea.